pkg/vendir/directory/staging_dir.go

```diff
@@ -104,7 +104,7 @@ func (d StagingDir) CopyExistingFiles(rootDir string, stagingPath string, ignore
 		}
 
 		// Move the file to the staging directory
-		err = copy(path, stagingPath)
+		err = copyFunc(path, stagingPath)
```

Comment:
Is there a copy function introduced in Go v1.20? 
nit: copyFunc -> copyFile

Reply:
Not sure when it got introduced but this is what golint is throwing::
```
pkg/vendir/directory/staging_dir.go:197:1: redefines-builtin-id: redefinition of the built-in function copy (revive)
```